SQF/dayz_code/compile/player_unlockDoor.sqf

```diff
@@ -2,39 +2,61 @@
 	DayZ Unlock Door
 	Usage: [_obj] call player_unlockDoor;
 	Made for DayZ Epoch please ask permission to use/edit/distrubute email vbawol@veteranbastards.com.
+	Modified for Zupa's DoorManagement.
 */
 private ["_display","_obj","_objectCharacterID"];
 
 if (!isNil "DZE_DYN_UnlockDoorInprogress") exitWith {localize "str_epoch_player_21" call dayz_rollingMessages;};
 
 DZE_DYN_UnlockDoorInprogress = true;
 
+// find display and check the door opening method
+_doorMethod = '';
+_displayCombo = findDisplay 41144;
+_displayEye = findDisplay 61144;
+if(!isNull _displayEye) then {_display = _displayEye; _doorMethod = "Eye";};
+if(!isNull _displayCombo) then {_display = _displayCombo; _doorMethod = "Combo";};
+
 if (!isNull dayz_selectedDoor) then {
 	if (!isNil 'KeyCodeTryTimer') then {
 		if (diag_tickTime > KeyCodeTryTimer) then {
 			KeyCodeTry = nil;
 			KeyCodeTryTimer = nil;
 		};
 	};
-
+	
 	_obj = dayz_selectedDoor; // our target
 	_notNearestPlayer = _obj call dze_isnearest_player;
-
+	
 	if (_notNearestPlayer) then {
 		// close display since another player is closer
-		_display = findDisplay 41144;
 		_display closeDisplay 3000;
 		localize "STR_EPOCH_ACTIONS_16" call dayz_rollingMessages;
 	} else {
 		// get object combination
 		_objectCharacterID 	= _obj getVariable ["CharacterID","0"];
-
+		
+		if(DZE_doorManagement) then {
+			_doorOwnerID = if(DZE_plotforLife) then { dayz_selectedDoor getVariable ["ownerPUID","0"]; } else { dayz_selectedDoor getVariable ["characterID","0"]; };
```

Comment:
Ooops. Of course. You renamed it while I merged doorManagement in. This one I must have missed.